### PR TITLE
LBSD-302 implement backend for autoupdates

### PR DESCRIPTION
**Pull request by ancafarcas**

### CODE REVIEW

**Comment by nistormihai** [comment]
:thumbsup:

File: liveblog/blogs/blogs.py [hunk 1]
```diff
@@ -34,13 +53,31 @@ class BlogsResource(Resource):
...
+            'type': 'integer'
         }
     }
 
+    item_methods = ['GET', 'PATCH', 'PUT', 'DELETE']
     privileges = {'GET': 'blogs', 'POST': 'blogs', 'PATCH': 'blogs', 'DELETE': 'blogs'}
 
 
-class BlogService(BaseService):
+def set_cid_on_blogs(item):
```
Comment (ioanpocol) [inline liveblog/blogs/blogs.py]: One key is used for all blogs. To create a key for every blog you can use something like this: key = 'blog_cid' + str(item.blog)

**Comment by ioanpocol** [comment]
:+1:

**Comment by vied12** [comment]
PR has moved here https://github.com/superdesk/liveblog/pull/3

**Pull request closed without merging**